packages/cid-verifier/wrangler.toml

```diff
@@ -64,3 +70,7 @@ IPFS_GATEWAY_TLD = 'ipfs.link.test'
 DEBUG = "true"
 ENV = "test"
 GOOGLE_EVALUATE_SAFE_CONFIDENCE_LEVELS = [ "CONFIDENCE_LEVEL_UNSPECIFIED", "SAFE" ]
+# These are for testing purposes
+# echo -n 'foo:foo' | base64
+# echo -n 'zoo:zoo' | base64
+BASIC_AUTH_TOKENS = [ "Zm9vOmZvbw==", "em9vOnpvbw==" ]
```

Comment:
Can we inject them in miniflare globals instead?

https://github.com/web3-storage/minibus/blob/7fe8372c49c697759a88d6e2725185c7a2fe99da/test/scripts/helpers.js#L3